### ISSUE TITLE
Calendar component: Screen read doesn't read out anything when changing the year

### ISSUE THREAD

**Issue by lukehumphrey** [OfficeDev/office-ui-fabric-react#7620]
Use Edge/Narrator on https://developer.microsoft.com/en-us/fabric#/components/Calendar
Tab to the year up/down arrows and press one to change the year
Expected: SR reads out useful confirmation of new year
Result: No SR

Logged by Wipro against Project's Roadmap feature, which leverages this contro.

**Comment by dzearing** [comment]
@lorejoh12 FYI
